DigitalLearningSolutions.Data/Helpers/DateHelper.cs

```diff
@@ -7,6 +7,8 @@
 
     public static class DateHelper
     {
+        public static string StandardDateFormat = "dd/MM/yyyy";

```

Comment:
We now have this standard date format in 2 places again (there is one in a helper in the web project too). Either we need to get rid of that one in favour of this one, or we need that one to reference this one.